### PR TITLE
#166787650 Display articles

### PR DESCRIPTION
**Pull request by gracelungu**
## What does this PR do?
This PR adds pages to display articles

#### Description of Task to be completed?
- Design the home page
- Design the user's article page
- Fetch articles
- Test the page with cypress and jest

#### How should this be manually tested?
`localhost:300/myarticles`
`localhost:300/`

#### Any background context you want to provide?

N/A

#### What are the relevant pivotal tracker stories?
[#166787650](https://www.pivotaltracker.com/n/projects/2321845/stories/166787650)

#### Screenshots (if appropriate)
image

image

#### Questions:

N/A

### CODE REVIEW

File: src/Components/Articles/Article.jsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+ *
+ * @param {*} props
+ * @returns {object} jsx
+ */
+const Article = props => {
+  let { list } = props;
+  list = list.reverse().slice(0, 10);
+
+  const articles = list.map(
+    ({ id, image, title, description, readTime, favoritesCount, Ratings, commentsCount, slug }) => {
```
Comment (Karlmusingo) [inline src/Components/Articles/Article.jsx]: In our backend database, the `images` is an array of all images used in the article. I think you should take the first image in the array and use it.

Comment (gracelungu) [reply src/Components/Articles/Article.jsx]: @Karlmusingo I have implemented the changes.

File: cypress/integration/home.spec.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
+describe('Home', () => {
+  it('Should render Home', () => {
+    cy.server();
+    cy.route({
+      method: 'GET', // Route all GET requests
+      url: '/api/articles', // that have a URL that matches '/articles/*'
```
Comment (giseleiradu) [inline cypress/integration/home.spec.js:6]: I would recommend making the comments clearer and fully contained.

**Review by giseleiradu — APPROVED** [review]

File: cypress/integration/home.spec.js (hunk 2 above)
Comment (giseleiradu) [reply cypress/integration/home.spec.js:6]: Also, you may need to check the CIs claims for a fix. Otherwise, good job!

**Review by Karlmusingo — APPROVED** [review]

**Pull request merged**